package.json

```diff
@@ -146,6 +146,7 @@
     "codecov": "^3.7.1",
     "core-js": "^3.7.0",
     "cross-env": "^6.0.3",
+    "cypress": "^12.4.0",
```

Comment:
I think we should update the cypress package in "optionalDependencies" @andrzejewsky

Reply:
I think you're right